### PR TITLE
Added telegram notification service

### PR DESCRIPTION
**Pull request by Dimagaa**

### CODE REVIEW

File: src/main/java/com/jvavateam/carsharingapp/notification/telegram/handlers/AboutUsHandler.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            jvavateam@gmail.com
+            
+            Members:
+            
+            Oleksandr Parkhomovskyi
+            Sofiya Kostashchuk
+            Artem Reizer
+            Ivan Fadieiev
+            Dmytro Martyshchuk
+            """;
```
Comment (IvanFadieiev) [inline src/main/java/com/jvavateam/carsharingapp/notification/telegram/handlers/AboutUsHandler.java:22]: Looks good!

File: src/main/java/com/jvavateam/carsharingapp/service/impl/SubscriptionServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    private final UserRepository userRepository;
+
+    @Override
+    @Transactional
+    public Optional<Subscription> create(SubscriptionDto subscriptionDto) {
+        Optional<User> user = userRepository
+                .findByEmail(subscriptionDto.userEmail());
+        return user.flatMap(value -> subscriptionRepository.findByChatId(subscriptionDto.chatId())
+                .or(() -> createNew(subscriptionDto, value)));
+
```
Comment (IvanFadieiev) [inline src/main/java/com/jvavateam/carsharingapp/service/impl/SubscriptionServiceImpl.java]: redundant empty line :)

File: src/main/resources/db/changelog/changes/09-create-subscriptions-table.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-subscriptions-table
+      author: admin
```
Comment (IvanFadieiev) [inline src/main/resources/db/changelog/changes/09-create-subscriptions-table.yaml]: it's better to follow our changeset style and use our own names

**Review by IvanFadieiev — CHANGES_REQUESTED** [review]
GJ! Some small things to change :)

**Review by IvanFadieiev — APPROVED** [review]
Good job!

**Review by holsoni — APPROVED** [review]
I haven't found anything looking bad(

**Pull request merged**